### PR TITLE
Add 8.2 version

### PR DESCRIPTION
**Pull request by dmetzler**
Hello, 

We've bundled our new 8.2 version.

### CODE REVIEW

File: library/nuxeo [hunk 1]
```diff
@@ -1,8 +1,9 @@
 # maintainer: Damien Metzler <dmetzler@nuxeo.com> @dmetzler
 
-latest: git://github.com/nuxeo/docker-nuxeo@09c50e490336f28caf1278c6bffaa02e79c0f4b4 8.1
-FT: git://github.com/nuxeo/docker-nuxeo@09c50e490336f28caf1278c6bffaa02e79c0f4b4 8.1
-8: git://github.com/nuxeo/docker-nuxeo@09c50e490336f28caf1278c6bffaa02e79c0f4b4 8.1
+latest: git://github.com/nuxeo/docker-nuxeo@c8f5d5bd4fedda1ee72eb29eac5fbb0b0a490155 8.2
+FT: git://github.com/nuxeo/docker-nuxeo@c8f5d5bd4fedda1ee72eb29eac5fbb0b0a490155 8.2
+8: git://github.com/nuxeo/docker-nuxeo@c8f5d5bd4fedda1ee72eb29eac5fbb0b0a490155 8.2
+8.2: git://github.com/nuxeo/docker-nuxeo@c8f5d5bd4fedda1ee72eb29eac5fbb0b0a490155 8.2
 8.1: git://github.com/nuxeo/docker-nuxeo@09c50e490336f28caf1278c6bffaa02e79c0f4b4 8.1
```
Comment (tianon) [inline library/nuxeo]: Any particular reason to keep this version listed here?  Removing it here will remove it from the "Supported" section on the Hub readme (and will prevent us from spending cycles rebuilding it on the official build server), but the tag will still be available to users who want it.  (See https://github.com/docker-library/official-images#library-definition-files for more detail on this.)

**Comment by tianon** [comment]
Changes LGTM: https://github.com/nuxeo/docker-nuxeo/compare/09c50e490336f28caf1278c6bffaa02e79c0f4b4...c8f5d5bd4fedda1ee72eb29eac5fbb0b0a490155

```diff
$ git diff -C -C -C 09c50e490336f28caf1278c6bffaa02e79c0f4b4...c8f5d5bd4fedda1ee72eb29eac5fbb0b0a490155
diff --git a/6.0/Dockerfile b/5.8/Dockerfile
similarity index 95%
copy from 6.0/Dockerfile
copy to 5.8/Dockerfile
index 530f99b..17cbc47 100644
--- a/6.0/Dockerfile
+++ b/5.8/Dockerfile
@@ -1,5 +1,5 @@
 # vim:set ft=dockerfile:
-FROM       java:8-jre
+FROM       java:7-jre
 MAINTAINER Nuxeo <packagers@nuxeo.com>
 
 # Create Nuxeo user
@@ -33,8 +33,8 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
     ghostscript \
  && rm -rf /var/lib/apt/lists/*
 
-ENV NUXEO_VERSION 6.0
-ENV NUXEO_MD5 b8cbc0b2858b0697a541be49feb24571
+ENV NUXEO_VERSION 5.8
+ENV NUXEO_MD5 258efadac41b2d8f8e6035b6acbf5a6d
 ENV NUXEO_HOME /opt/nuxeo/server
 
 # Add distribution
diff --git a/docker-entrypoint.sh b/5.8/docker-entrypoint.sh
similarity index 100%
copy from docker-entrypoint.sh
copy to 5.8/docker-entrypoint.sh
diff --git a/6.0/Dockerfile b/8.2/Dockerfile
similarity index 96%
copy from 6.0/Dockerfile
copy to 8.2/Dockerfile
index 530f99b..c306a7c 100644
--- a/6.0/Dockerfile
+++ b/8.2/Dockerfile
@@ -33,8 +33,8 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
     ghostscript \
  && rm -rf /var/lib/apt/lists/*
 
-ENV NUXEO_VERSION 6.0
-ENV NUXEO_MD5 b8cbc0b2858b0697a541be49feb24571
+ENV NUXEO_VERSION 8.2
+ENV NUXEO_MD5 13cbc55be36823bec99482c651c1cbc0
 ENV NUXEO_HOME /opt/nuxeo/server
 
 # Add distribution
diff --git a/docker-entrypoint.sh b/8.2/docker-entrypoint.sh
similarity index 100%
copy from docker-entrypoint.sh
copy to 8.2/docker-entrypoint.sh
```

File: library/nuxeo (hunk 1 above)
Comment (dmetzler) [inline library/nuxeo]: Not any particular reason. 8.1 and 7.4 should not evolve as these are fastrack versions. I will remove them from the build file

**Comment by dmetzler** [comment]
@tianon i've removed old FT versions. Ok to merge ?

**Comment by yosifkit** [comment]
LGTM, Build test of #1645; e4d0979 (`nuxeo`):

```console
$ bashbrew build "nuxeo"
Cloning nuxeo (git://github.com/nuxeo/docker-nuxeo) ...
Processing nuxeo:latest ...
Processing nuxeo:FT ...
Processing nuxeo:8 ...
Processing nuxeo:8.2 ...
Processing nuxeo:LTS-2015 ...
Processing nuxeo:LTS ...
Processing nuxeo:7.10 ...
Processing nuxeo:7 ...
Processing nuxeo:LTS-2014 ...
Processing nuxeo:6 ...
Processing nuxeo:6.0 ...
$ bashbrew list --uniq "$url" | xargs test/run.sh
testing nuxeo:latest
	'utc' [1/6]...passed
	'cve-2014--shellshock' [2/6]...passed
	'no-hard-coded-passwords' [3/6]...passed
	'override-cmd' [4/6]...passed
	'nuxeo-conf' [5/6]...passed
	'nuxeo-basics' [6/6]....passed
testing nuxeo:LTS-2015
	'utc' [1/6]...passed
	'cve-2014--shellshock' [2/6]...passed
	'no-hard-coded-passwords' [3/6]...passed
	'override-cmd' [4/6]...passed
	'nuxeo-conf' [5/6]...passed
	'nuxeo-basics' [6/6]....passed
testing nuxeo:LTS-2014
	'utc' [1/6]...passed
	'cve-2014--shellshock' [2/6]...passed
	'no-hard-coded-passwords' [3/6]...passed
	'override-cmd' [4/6]...passed
	'nuxeo-conf' [5/6]...passed
	'nuxeo-basics' [6/6]....passed
```

**Pull request merged**